util/packaging/homebrew/chapel-main.rb

```diff
@@ -17,7 +17,7 @@ class Chapel < Formula
   end
 
   depends_on "gmp"
-  depends_on "python@3.10"
+  depends_on "python@3.9"
```

Comment:
Can you put a comment explaining the situation here?

Reply:
Oh goodness - yes of course.  Thanks!